sphinx_copybutton/__init__.py

```diff
@@ -68,6 +69,7 @@ def setup(app):
     app.add_config_value("copybutton_here_doc_delimiter", "", "html")
     app.add_config_value("copybutton_image_path", "copy-button.svg", "html")
     app.add_config_value("copybutton_selector", "div.highlight pre", "html")
+    app.add_config_value("copybutton_exclude", ".linenos", "html")
```

Comment:
```suggestion
    app.add_config_value("copybutton_exclude", ".linenos, .gp", "html")
```

Wait, does this work to select multiple?  In the end it gets passed to the filterText function.